tweak(natives/hud): add description and example for SET_BLIP_COLOUR (#1166)
citizenfx/natives: HUD/SetBlipColour.md

void SET_BLIP_COLOUR(Blip blip, int color);
```

> 

Reviewer: ```suggestion
```

void SET_BLIP_COLOUR(Blip blip, int color);
```

> Sets the color of a blip. The color can also be a hex if you want a color that's not included in the list above.

Reviewer: ```suggestion
```

## Parameters

> * **blip**: A blip handle

Reviewer: ```suggestion
* **blip**: The blip handle to set the color of
```

> * **color**: See [blip colors here](https://docs.fivem.net/docs/game-references/blips/#blip-colors)

Reviewer: ```suggestion
* **color**: This can be a hex color code, or a [blip color](https://docs.fivem.net/docs/game-references/blips/#blip-colors).
```
